docs/examples/FormHorizontal.js

```diff
@@ -0,0 +1,37 @@
+const formInstance = (
+  <Form horizontal>
+    <FormGroup controlId="formHorizontalEmail">
+      <Col componentClass={ControlLabel} xs={2}>
+        Email
+      </Col>
+      <Col xs={10}>
+        <FormControl type="email" placeholder="Email" />
+      </Col>
+    </FormGroup>
+
+    <FormGroup controlId="formHorizontalPassword">
+      <Col componentClass={ControlLabel} xs={2}>
```

Comment:
I'm okay with this API I think. Doesn't this come up just as much for building e.g. left tabs? You could write this as

```js
<Col xs={2}>
  <ControlLabel>Password</ControlLabel>
</Col>
```

which just adds an extra `<div>` that doesn't do any harm; this is just closer to the upstream TWBS example.